kornia/feature/loftr/utils/geometry.py

```diff
@@ -0,0 +1,54 @@
+import torch
+
+
+@torch.no_grad()
+def warp_kpts(kpts0, depth0, depth1, T_0to1, K0, K1):
```

Comment:
not now but this should go as core function in `kornia.geometry`. Add a todo or something to be aware about that